src/components/Facility/FacilityForm.tsx

```diff
@@ -39,48 +41,50 @@ import {
 
 import routes from "@/Utils/request/api";
 import mutate from "@/Utils/request/mutate";
+import query from "@/Utils/request/query";
 import { parsePhoneNumber } from "@/Utils/utils";
 import OrganizationSelector from "@/pages/Organization/components/OrganizationSelector";
 import { BaseFacility } from "@/types/facility/facility";
+import { Organization } from "@/types/organization/organization";
+
+import { FacilityModel } from "./models";
 
 const facilityFormSchema = z.object({
-  facility_type: z.string().min(1, "Facility type is required"),
-  name: z.string().min(1, "Name is required"),
+  facility_type: z.string().min(1, t("facility_type_required")),
+  name: z.string().min(1, t("name_is_required")),
   description: z.string().optional(),
   features: z.array(z.number()).default([]),
-  pincode: z.string().refine(validatePincode, "Invalid pincode"),
-  address: z.string().min(1, "Address is required"),
+  pincode: z.string().refine(validatePincode, t("invalid_pincode")),
+  geo_organization: z.string().min(1, t("organization_required")),
```

Comment:
i18n's `t` func. imported might work in local dev server, but does it work in prod. build?